manifest.json

```diff
@@ -10,12 +10,37 @@
         "targetDirPrefix": "backports.functools_lru_cache-"
       },
       "bleach": {
-        "version": "1.2.2",
-        "downloadFormat": "zip",
-        "url": "https://github.com/jsocol/bleach/archive/v1.2.2.zip",
+        "version": "3.1.0",
+        "downloadFormat": "tar",
+        "url": "https://github.com/mozilla/bleach/archive/v3.1.0.tar.gz",
+        "tarRootDirPrefix": "bleach-",
         "rootDirPrefix": "bleach-",
         "targetDirPrefix": "bleach-"
       },
+      "six": {
+        "version": "1.12.0",
+        "downloadFormat": "tar",
+        "url": "https://github.com/benjaminp/six/archive/1.12.0.tar.gz",
+        "tarRootDirPrefix": "six-",
+        "rootDirPrefix": "six-",
+        "targetDirPrefix": "six-"
+      },
+      "python-webencodings": {
+        "version": "0.5.1",
+        "downloadFormat": "tar",
+        "url": "https://github.com/gsnedders/python-webencodings/archive/v0.5.1.tar.gz",
+        "tarRootDirPrefix": "python-webencodings-",
+        "rootDirPrefix": "python-webencodings-",
+        "targetDirPrefix": "python-webencodings-"
+      },
+      "callbacks": {
+        "version": "0.3.0",
+        "downloadFormat": "tar",
+        "url": "https://files.pythonhosted.org/packages/eb/15/c71150189ea1cfd04a5853a40e2e074ec777755bd3924980fe51e5c6dd44/callbacks-0.3.0.tar.gz",
```

Comment:
Can we use the github version?

Reply:
They haven't yet created a GitHub release for version 0.3.0. I have [raised an issue](https://github.com/14rcole/callbacks/issues/16) on the repository for the same.
 Up until then, is it fine if we continue using the PyPi link?